tests/test_prog

```diff
@@ -1,36 +1,37 @@
 #!/usr/bin/env matrix
-tests:
-- name: deployment
-  description: Deploy with Glitches
-  rules:
-    - do:
-        action: matrix.plugins.deploy.deploy
-        entity: cs:bundle/wiki-simple
-        version: current
-    - do:
-        action: tests.chaos.chaos
-    - do:
-        action: health
-      until: chaos.complete
-      "on": state.change
-      after: deploy
-- name: traffic
-  description: Traffic in the face of Chaos
-  rules:
-    - do:
-        action: matrix.plugins.deploy.deploy
-        entity: cs:bundle/wiki-simple
-        version: current
-    - do:
-        action: tests.traffic.test_traffic
-      until: chaos.complete
-      after: deploy
-    - do:
-        action: tests.chaos.chaos
-      while: test_traffic
-    - do:
-        action: health
-      periodic: 5
-      "on": state.change
-      until: chaos.complete
-      after: deploy
+"tests":
+- "name": deployment
+  "description": Deploy with Glitches
+  "rules":
+    - "do":
+        "action": matrix.plugins.actions.deploy
+        "entity": "cs:bundle/wiki-simple"
+        "version": current
+    - "after": deploy
+      "periodic": 5
+      "do":
+        "action": matrix.plugins.actions.health
+      "until": glitch.complete
+    - "after": deployment.healthy
```

Comment:
Still working on generalizing support for this type of thing. After that is done I'll have some merge conflicts with the above but I'll deal with those.